### PR TITLE
machine_configuration: Update readme

### PR DESCRIPTION
**Pull request by hershpa**
Disable the in-tree i915 and intel_vsec configuration so KMM can modprobe the
OOT drivers.
This is the short-term solution. In the long-run, the in-tree drivers unload should be handled by KMM operator.

Signed-off-by: Hersh Pathak hersh.pathak@intel.com

### CODE REVIEW

**Comment by uMartinXu** [comment]
see my comments above.

File: machine_configuration/README.md [hunk 1]
```diff
@@ -70,22 +70,21 @@ intel-dgpu   rendered-intel-dgpu-58fb5f4d72fe6041abb066880e112acd   True      Fa
...
 ## Verification
 Navigate to the node terminal on the web console (Compute -> Nodes -> Select a node -> Terminal). Run the following commands in the terminal.
 ```
 $ chroot /host
-$ lsmod | grep ast
+$ lsmod | grep i915
+$ lsmod | grep intel_vsec
 ```
-Ensure that ast driver is not loaded.
+Ensure that i915 and intel_vsec driver is not loaded.
```
[the remark this answers is not in the document]
Comment (hershpa) [reply machine_configuration/README.md:87]: Got it

File: machine_configuration/README.md [hunk 2]
```diff
@@ -70,22 +70,21 @@ intel-dgpu   rendered-intel-dgpu-58fb5f4d72fe6041abb066880e112acd   True      Fa
...
 
-# Disable conflicting driver
-Run the command shown below to disable the loading of a potential conflicting driver, such as `ast` driver.
-
-**Note**: The `i915` driver depends on a ported `drm` module. Some other drivers, such as ast that depends on in-tree drm module might have a compatibility issue. The known issue will be resolved on i915 driver for RHEL `9.x`, which will be used for RHOCP `4.13`. 
+# Disable in-tree drivers
+Run the command shown below to disable the loading of in-tree drivers 'i915' and 'intel_vsec'.
 ```
-$ oc apply -f https://raw.githubusercontent.com/intel/intel-technology-enabling-for-openshift/main/machine_configuration/100-intel-dgpu-machine-config-disable-ast.yaml
+$ oc apply -f https://raw.githubusercontent.com/intel/intel-technology-enabling-for-openshift/main/machine_configuration/100-intel-dgpu-machine-config-disable-i915.yaml
```
[the remark this answers is not in the document]
Comment (hershpa) [reply machine_configuration/README.md:76]: Agreed

**Comment by hershpa** [comment]
Updated PR, thanks for the feedback @uMartinXu!

**Comment by uMartinXu** [comment]
LGTM

**Pull request merged**